tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -788,6 +789,36 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         this.set('_isReloadDisabled', dashboard.disableReload);
       },
 
+      _renderPluginIFrame(container, selectedDashboard, loadingMechanism) {
+        if (this._inColab) {
+          const templateContent = this.$.colabNotSupported.content;
+          const node = document.importNode(templateContent, true);
+          this.scopeSubtree(node);
+          container.appendChild(node);
+          return;
```

Comment:
In this code path, we don’t set `x.id = 'dashboard';` for any `x`; just
want to confirm that that’s okay and doesn’t cause anything to crash
(e.g., reloading)?

Reply:
Added the id. Btw, the `id = 'dashboard'` is not effectively useful since we are not rendering an instance of a plugin and this conditional will never be truthy: https://github.com/tensorflow/tensorboard/blob/ff05d1af8d118d0add237766980d31163a94c40b/tensorboard/components/tf_tensorboard/tf-tensorboard.html#L1000-L1001.

Regardless, I added the `id` so future readers don't have to remember/understand every flow.

Reply:
Got it; thanks!